Neos.Flow/Classes/Package/Package.php

```diff
@@ -97,12 +97,11 @@ class Package implements PackageInterface
      * @param string $composerName
      * @param string $packagePath Absolute path to the location of the package's composer manifest
      * @param array $autoloadConfiguration
-     * @throws Exception\InvalidPackageKeyException
      */
     public function __construct($packageKey, $composerName, $packagePath, array $autoloadConfiguration = [])
     {
         $this->autoloadConfiguration = $autoloadConfiguration;
-        $this->packagePath = Files::getNormalizedPath($packagePath);
+        $this->packagePath = $packagePath;
```

Comment:
That's a leftover from the oooooold package manager of old where this might have not been normalised elsewhere. These days a package object is always instantiated via PackageFactory which normalises on it's own.